src/deepak/FindPosNegZeroInUserInputArray.java

```diff
@@ -0,0 +1,46 @@
+package deepak;
+
+import java.util.Scanner;
+
+public class FindPosNegZeroInUserInputArray {
+	
+	int posNumbersCount, negNumbersCount, zeroCount;
+
+	int positiveNumberInArray(int[] input) {
+		
```

Comment:
this 3 method can we merge in one with if else condition
if(arr[i]>0)
  pCnt++;
else if(arr[i]<0)
  nCnt++;
else
  zCnt++;